.devcontainer/dockerBuild.sh

```diff
@@ -0,0 +1,14 @@
+#environment setup
+sudo chmod -R a+rwx /etc/bash.bashrc
+echo "export PATH=\"/home/node/.cache/pypoetry/virtualenvs/gbstats-vabhrsvx-py3.7/bin:$PATH:$HOME/.poetry/bin\"" >> /etc/bash.bashrc
+
+#poetry installation
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
+
+#needed for the current shell session
+export PATH="/home/node/.cache/pypoetry/virtualenvs/gbstats-vabhrsvx-py3.7/bin:$PATH:$HOME/.poetry/bin"
```

Comment:
kind of a hack but `. /etc/bash.bashrc` breaks and source `/etc/bash.bashrc` doesn't work

Reply:
I think if we get rid of `yarn dev` from this script, we might not need this since the user will need to open a new shell first before running it.